lookout/style/format/feature_utils.py

```diff
@@ -36,6 +36,19 @@ def __init__(self, value: str, start: Position, end: Position,
         self.y = y
         self.path = path
 
+    @property
+    def is_accumulated_indentation(self) -> bool:
```

Comment:
I prefer to have a property because if someone changes something in VirtualNode (it implied to be read-only, but you never know) `is_accumulated_indentation` flag can be broken.

Reply:
The proper fix would be to make VirtualNodes readonly then (NamedTuple).

Reply:
let's made them read-only with properties. WDYT?

Reply:
I don't think there is a point in having readonly properties instead of a tuple. It'll be less straight-forward and less efficient.

Reply:
So there are some points against converting it to a tuple:
1. I do not like how the code looks like if you Extend NamedTuple class. It is harder to maintain and you loose support for PyCharm hints 
2. You are able to use `virtual_node[1]` and I think it is a bad option.
3. I am sure the efficiency gap is super small compare to all our efficiency problems.

I have an alternative solution to mark VirtualNodes as nodes with an accumulated indentation in the parser and do not do it here because you actually know it only there. So we avoid properties (but I think we actually will add it at some moment https://github.com/src-d/style-analyzer/issues/161) and converting the class to NamedTuple. 

Or keep it as is and add more read-only properties :)

WDYT?

Reply:
After some more thoughts it will be a pain to convert to namedtuples since we have a multi-pass algorithm already in the parser.

We should therefore not enforce anything but stick to the convention that virtualnode is readonly apart from the parsing phase. Therefore we should go back to my first comment I think and just set the corresponding attribute directly.